mountpoint-s3-client/tests/put_object.rs

```diff
@@ -167,9 +169,51 @@ async fn test_put_object_abort() {
 
     drop(request); // Drop without calling complete().
 
-    // Allow for the AbortMultipartUpload to complete.
+    // Try to wait a while for the async abort to complete. For the larger object, this might be
```

Comment:
This seems to still not be enough: https://github.com/awslabs/mountpoint-s3/actions/runs/8134885229/job/22228500040?pr=794#step:8:1228

Reply:
I'm still messing with it to confirm, but it looks like a CRT-side issue — the AbortMultipartUpload can race with in-flight UploadParts, which might succeed after the Abort succeeds and so re-create the upload. I'm not sure if that's worth fixing here (since we're not actually changing how PUT works), so I might just disable the test for large objects.